src/app/stop-the-breed/dialog/dialog.component.html

```diff
@@ -0,0 +1,23 @@
+<div *ngIf=" !data.tutorial">
+
+  <div mat-dialog-content>
+    Oh Yeah!! You just removed <b> {{ data.name }} :D </b>.
```

Comment:
No hardcoded strings. Please put them in english.json file and use it here using the language service.